src/sentry/runner/commands/run.py

```diff
@@ -328,10 +328,10 @@ def cron(**options: Any) -> None:
     help="A file to touch roughly every second to indicate that the consumer is still alive. See https://getsentry.github.io/arroyo/strategies/healthcheck.html for more information.",
 )
 @click.option(
-    "--enable-dlq",
+    "--enable-dlq/--disable-dlq",
```

Comment:
Looks good but I don't understand this part.  What does the slash mean in this context?  And why would we want the negation to share the same parameter?

Reply:
Google is my friend: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags